src/Appwrite/Platform/Modules/Functions/Workers/Builds.php

```diff
@@ -1077,57 +1072,80 @@ protected function buildDeployment(
             Console::success("Build id: $deploymentId created");
 
             /** Set auto deploy */
+            $activateBuild = true;
             if ($deployment->getAttribute('activate') === true) {
```

Comment:
I dont like `activateBuild` being `true` by default.
I am worried due to bug, we might be activating preview deployment, which is terrible experience.

To be safe, let's update this to be `false` by default, and only set to `true` if deployment has activate true, and it's not having race condition issue